core/src/main/java/org/kiwix/kiwixmobile/core/utils/SharedPreferenceUtil.kt

```diff
@@ -160,10 +173,21 @@ class SharedPreferenceUtil @Inject constructor(val context: Context) {
       _textZooms.offer(textZoom)
     }
 
+  fun getActualPath(path: String): String {
+    var s = path
```

Comment:
This name of the function doesn't justify that the function actually does. A better function name would be nice.

Reply:
rename to getPublicDirectoryPath